common/src/main/java/party/lemons/biomemakeover/level/feature/OrangeGlowshroomFeature.java

```diff
@@ -31,11 +28,14 @@ public boolean place(FeaturePlaceContext<ProbabilityFeatureConfiguration> featur
         RandomSource random = featurePlaceContext.random();
         WorldGenLevel level = featurePlaceContext.level();
         BlockPos pos = featurePlaceContext.origin();
-        int xOffset = random.nextInt(8) - random.nextInt(8);
```

Comment:
It seems that the purpose of this spread is to give the placements a bit of generation range into the neighboring ocean biome.